net-print/kyocera-1x2x-mfp-driver/kyocera-1x2x-mfp-driver-1.1203-r1.ebuild

```diff
@@ -0,0 +1,129 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Printer descriptions (PPDs) and filters for Kyocera 1x2x MFP"
+HOMEPAGE="http://www.kyoceradocumentsolutions.eu"
+SRC_URI="LinuxDrv_${PV}_FS-1x2xMFP.zip"
+
+LICENSE="kyocera-mita-ppds GPL-2"
+SLOT="0"
+
+KEYWORDS="-* ~amd64"
+
+IUSE_L10N=(en ar cs de el es fr he hu it ko pl pt ro ru th tr vi zh-CN zh-TW)
+IUSE="+rastertokpsl-fix ${IUSE_L10N[@]/#/l10n_}"
+REQUIRED_USE="|| ( ${IUSE_L10N[@]/#/l10n_} )"
+
+RDEPEND="net-print/cups"
+DEPEND="app-arch/unzip"
+RESTRICT="fetch mirror"
+QA_PREBUILT="/usr/libexec/cups/filter/rastertokpsl"
+
+get_tarball_name() {
+	# Note the capitalization inconsistency. Don't "fix" that.
+	case "$1" in
+		ar) echo arabic ;;
+		cs) echo czech ;;
+		de) echo German ;;
+		el) echo greek ;;
+		en) echo English ;;
+		es) echo Spanish ;;
+		fr) echo French ;;
+		he) echo hebrew ;;
+		hu) echo hungarian ;;
+		it) echo Italian ;;
+		ko) echo Korean ;;
+		pl) echo polish ;;
+		pt) echo Portuguese ;;
+		ro) echo romanian ;;
+		ru) echo russian ;;
+		th) echo thai ;;
+		tr) echo turkish ;;
+		vi) echo vietnamese ;;
+		zh-CN) echo simplified ;;
+		zh-TW) echo traditional ;;
+		*) die ;;
+	esac
+}
+
+pkg_nofetch() {
+	einfo "Please, navigate your browser to the following URL and manually"
+	einfo "download the file named '${A}', then put it into '${DISTDIR}'."
+	einfo "http://www.kyoceradocumentsolutions.eu/index/service/dlc.false.driver.FS1025MFP._.EN.html"
```

Comment:
```
 * Please, navigate your browser to the following URL and manually
 * download the file named 'LinuxDrv_1.1203_FS-1x2xMFP.zip', then put it into '/tmp/tmprn9z_egz/portage/net-print/kyocera-1x2x-mfp-driver-1.1203-r1/distdir'.
```

Not very helpful directory, i.e. you are using `DISTDIR` the way it's not supposed to be used.

Reply:
Also, https here. Especially that the site redirects.